vars/piperPipelineStageMavenStaticCodeChecks.groovy

```diff
@@ -0,0 +1,32 @@
+import com.sap.piper.ConfigurationLoader
+import com.sap.piper.GenerateStageDocumentation
+import com.sap.piper.QualityCheck
+import com.sap.piper.ReportAggregator
+
+import static com.sap.piper.Prerequisites.checkScript
+
+import groovy.transform.Field
+
+@Field String STEP_NAME = getClass().getName()
+
+@Field Set GENERAL_CONFIG_KEYS = []
+@Field STAGE_STEP_KEYS = [
+    /** Execute static code checks for Maven based projects. This stage enforces SAP Cloud SDK specific PND rulesets as well as SpotBugs include filter.  */
+    'mavenExecuteStaticCodeChecks'
```

Comment:
Do you know what the effect of STAGE_STEP_KEYS  is?
Usually STAGE_STEP_KEYS is filled if you can decide which checks are executed. This is not the case here and thus I would remove this line.